sdcm/cluster_baremetal.py

```diff
@@ -16,9 +16,9 @@ class NodeIpsNotConfiguredError(Exception):
 
 class PhysicalMachineNode(cluster.BaseNode):
 
-    def __init__(self, name, public_ip, private_ip, credentials, base_logdir=None, node_prefix=None):
+    def __init__(self, name, public_ip, private_ip, credentials, base_logdir=None, node_prefix=None, baremetal_username=None):
         ssh_login_info = {'hostname': None,
-                          'user': credentials.name,
+                          'user': baremetal_username if baremetal_username else credentials.name,
```

Comment:
you need to use standard credentials, no need to create specific user creds for Physical cluster